financial-templates-lib/Robo-Caller.js

```diff
@@ -0,0 +1,82 @@
+require("dotenv").config();
+const { Logger } = require("./Logger");
```

Comment:
This may be stupid, but do recursive includes work in javascript? If you were to `require` this file in the `Logger`, would that break?

Reply:
not sure tbh. but here I'm not importing anything recursively so should be fine.

Reply:
After playing with this a bit more I think you are 100% correct that this ends up being recursive. I'm going to remove all logging from this script for now.

Reply:
My guess from articles like this: https://medium.com/visual-development/how-to-fix-nasty-circular-dependency-issues-once-and-for-all-in-javascript-typescript-a04c987cf0de is that circular dependencies don't work. I think circular dependency/include is the right term. Sorry if my language was confusing.